theme/templates/includes/access-row.html

```diff
@@ -1,133 +1,89 @@
 {% load hydroshare_tags %}
 
-{% for u in entries %}
-<tr id="row-id-{{ u.pk }}"
-    {% if access != "Is owner" %}
-    {% elif entries|length == 1 or not is_owner_user or u.pk == quota_holder.pk %}
-    class="hide-actions"
-    {% endif %}>
+<tr v-for="(user, index) in this.users"
+    :class="{ 'hide-actions': selfAccessLevel !== 'owner' && (user.access == 'owner' && hasOnlyOneOwner || user.id === quotaHolder.id), loading: user.loading}">
     <td>
         <table class="user-scope">
-            {% if access_type == "user" %}
-            <tr>
+            <tr v-if="user.user_type === 'user'">
                 <td>
```

Comment:
for here and all your v-if v-<conditionals> did you ensure you thought through 'truthy' state for all possible outcomes undefined, empty string, etc.?

Reply:
I think those edge cases are not possible with our current API, so there are no edge cases handlers here.